src/mlpack/tests/main_tests/softmax_regression_test.cpp

```diff
@@ -40,18 +40,17 @@ struct SoftmaxRegressionTestFixture
   }
 };
 
-BOOST_FIXTURE_TEST_SUITE(SoftmaxRegressionMainTest,
-                         SoftmaxRegressionTestFixture);
-
 /**
  * Ensure that we get desired dimensions when both training
  * data and labels are passed.
  */
-BOOST_AUTO_TEST_CASE(SoftmaxRegressionOutputDimensionTest)
+TEST_CASE_METHOD(
+    SoftmaxRegressionTestFixture,
+    "SoftmaxRegressionOutputDimensionTest", "SoftmaxRegressionMainTest")
```

Comment:
```suggestion
    "SoftmaxRegressionOutputDimensionTest",
    "[SoftmaxRegressionMainTest][BindingsTest]")
```
I think we should add the extra tag for bindings tests too. :+1: